### ISSUE TITLE
orderby on posts not working 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#201]
```
What steps will reproduce the problem? (Be specific)

Unsure... see this post: 
http://wordpress.org/support/topic/plugin-custom-content-type-manager-after-upgr
ading-orderby-on-posts-not-working-anymore
```

Original issue reported on code.google.com by `fireproofsocks` on 7 Oct 2011 at 6:03

**Comment by GoogleCodeExporter** [comment]
```
There are several options available for this:
 * 'none' - No order (available with Version 2.8).
 * 'ID' - Order by post id. Note the captialization.
 * 'author' - Order by author.
 * 'title' - Order by title.
 * 'date' - Order by date.
 * 'modified' - Order by last modified date.
 * 'parent' - Order by post/page parent id.
 * 'rand' - Random order.
 * 'comment_count' - Order by number of comments (available with Version 2.9).
 * 'menu_order' - Order by Page Order. Used most often for Pages (Order field in the Edit Page Attributes box) and for Attachments (the integer fields in the Insert / Upload Media Gallery dialog), but could be used for any post type with distinct 'menu_order' values (they all default to 0).
 * 'meta_value' - Note that a 'meta_key=keyname' must also be present in the query. Note also that the sorting will be alphabetical which is fine for strings (i.e. words), but can be unexpected for numbers (e.g. 1, 3, 34, 4, 56, 6, etc, rather than 1, 3, 4, 6, 34, 56 as you might naturally expect).
 * 'meta_value_num' - Order by numeric meta value (available with Version 2.8). Also note that a 'meta_key=keyname' must also be present in the query. This value allows for numerical sorting as noted above in 'meta_value'.
```

Original comment by `fireproofsocks` on 12 Oct 2011 at 6:54

**Comment by GoogleCodeExporter** [comment]
```
This has been fixed.  See issue 213. 

Committed revision 450176.

Will be released in 0.9.4.3
```

Original comment by `fireproofsocks` on 12 Oct 2011 at 8:00
- Changed state: **Fixed**
